### DISCUSSION TITLE
Custom Action Icons not displaying

### DISCUSSION THREAD

**Discussion by FlushCut** [UnchartedBull/OctoDash#3270] [Q&A]
Octodash is installed and working on a RPI 4 with a Hyperpixel 4.0 Square. 

Seven Custom Action buttons have been added, and they function correctly. The outlines for the Icons appear on the Control Screen in the selected color. The issue that I am having is that the Icons will not display, only the outline appears. 

Icons were temporarily assigned to the new buttons which were assigned to the six standard buttons. Those Icons displayed correctly.

The icons were selected from the FontAwesome Solid Free list:

arrow-up-long
arrow-down-long
arrow-turn-up
arrow-turn-down
arrows-to-circle
angles-up
angles-down

Any help will be appreciated. This has also been my intro to Raspberry Pi and Linux.

**Comment by jneilliii** [comment]
You might try my octodash companion plugin to see if they show the same way there.

**Reply by FlushCut** [reply]
Thank you for the suggestion. That works. The plugin works fantastic.

I wish there was a plugin for adjusting the tram, but I believe that is in the works for a future release.  In the meantime, just doing what I can to eliminate the need for the Creality display and just use Octodash. I don't want to be dependent on grabbing a Surface to take by the printer.

I really appreciate the help.
